plugins/fileservices/fileservices.cpp

```diff
@@ -2750,9 +2750,6 @@ static void checkExternalFileRights(ICodeContext *ctx,const char *scope,bool rd,
 static void checkExternalFilePath(ICodeContext *ctx,IPropertyTree *plane,const char *host,
     const char *path,bool rd,bool wr,RemoteFilename &rfn)
 {
-    if (containsRelPaths(path)) //Detect a path like: a/../../../f
-        throw makeStringExceptionV(-1,"Invalid file path %s",path);
-
```

Comment:
@wangkx - Now the rogue DeleteExternalFile's are removed (commented out), that was the root cause of the regression suite failure, is this movement of this relative check correct? 
If not should move back.
If it is, when does it cause failures?

Reply:
The relative check has been moved back to here.